k8s/helmfile/env/production/api.values.yaml.gotmpl

```diff
@@ -44,6 +44,8 @@ ingress:
   {{- end }}
 
 wbstack:
+  qsBatchesMarkFailedAfter: 3
+  qsBatchesRetryAfter: 'PT300S'
```

Comment:
If I'm not mistaken, in the chart these are named starting with `qsBatch` (vs `qsBatches`) so one of them needs to change

Reply:
also `RetryAfter` vs `PendingTimeout`